test/verify/check-docker-storage

```diff
@@ -124,10 +124,11 @@ class TestDockerStorage(MachineCase):
                     'MIN_DATA_SIZE=80M\n')
 
         def check_loopback(val):
+            # We pass the output through logger to diagnose flakey issues
             if val:
-                m.execute("losetup -l -O BACK-FILE | grep -q /var/lib/docker")
+                m.execute("losetup -l -O BACK-FILE | logger -s 2>&1 | grep -q /var/lib/docker")
```

Comment:
I'd suggest to also capture `losetup`'s error messages, they are even more interesting for logging than its regular stdout. But we must not let this affect the grep. How about something like
```sh
O=$(losetup -l -O BACK_FILE) 2>&1 | logger; echo "$O" | grep -q /var/lib/docker
```